deploy/olm-catalog/ibm-monitoring-grafana-operator/1.12.0/ibm-monitoring-grafana-operator.v1.12.0.clusterserviceversion.yaml

```diff
@@ -277,6 +277,12 @@ spec:
                 - update
                 - patch
                 - delete
+            - apiGroups:
+                - apps
+              resources:
+                - daemonsets
+              verbs:
+                - delete
```

Comment:
why was it required before and why this is no longer needed for removing the old deployments?

Reply:
i have included `daemonset` resource with `delete` verb because as part of cleanup we need to enable permission to delete nodeexporter daemonset